### PR TITLE
Makman2/nextgen3-7

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

File: tests/core/DependencyTrackerTest.py [hunk 1]
```diff
@@ -110,6 +110,32 @@ def test_get_all_dependencies(self):
...
+
+        self.assertEqual(uut.dependencies, set())
+
+        uut.add(1, 2)
+        uut.add(2, 3)
+        uut.add(3, 4)
+        uut.add(3, 5)
+        uut.add(2, 5)
+
+        self.assertEqual(uut.dependencies, {1, 2, 3})
```
Comment (Nosferatul) [inline tests/core/DependencyTrackerTest.py]: hmm, tests for the iterator?

Comment (Makman2) [inline tests/core/DependencyTrackerTest.py]: I think the doctests are enough. Do you want to have some specific corner case tested?

**Comment by sils** [comment]
commit message "Add dependants property" typo in `dependants` that's not a word

**Comment by Makman2** [comment]
@sils it is. The guy who is dependent -> dependant (dependent is also possible in American English)

File: tests/core/DependencyTrackerTest.py (hunk 1 above)
Comment (Nosferatul) [inline tests/core/DependencyTrackerTest.py]: sorry for the late response, it's fine :+1:

**Comment by Udayan12167** [comment]
ack https://github.com/coala/coala/pull/4136/commits/f0cc0980d6fb4c0a842517a54ce6314cd4c76a22

**Comment by Udayan12167** [comment]
ack https://github.com/coala/coala/pull/4136/commits/79dc8d5036369e396c3efc8ad0391f8fd62cfc68

**Comment by Udayan12167** [comment]
@rultor merge

**Comment by Makman2** [comment]
reack 1340b1b 28d1652 5a82d98

**Comment by Makman2** [comment]
Hm `review/gitmate/pr` seems to hang, repushing.

**Comment by Makman2** [comment]
reack 2922b4a

**Comment by Makman2** [comment]
reack 2922b4a

**Comment by Makman2** [comment]
@rultor merge
